C/plugins/storage/sqlite/connection.cpp

```diff
@@ -943,6 +943,10 @@ int		col = 0;
 		raiseError("update", "Failed to parse JSON payload");
 		return -1;
 	}
+	else if (document.HasMember("statsUpdate"))
```

Comment:
We are in the general table update routine, this should be a more generic bulk update that can be applied to all tables.

Can we make this so it fits in with the existing, we need to keep this generic for all tables. We should look at how we can define a set of updates